### PR TITLE
Model With 3 Background Noises

### PR DESCRIPTION
**Pull request by lukebushur**
Created the final version of my model that performs speech enhancement.
- Jupyter notebook and python file of code. I tried to keep the code modularized to reduce code duplication and provide ease-of-use for other team members
- The model is included as a .keras file. It was trained on approximately 17 hours of data. The model is trained on audio that has 1 of 3 types of background noises: traffic, cafe sounds (people talking), and vacuum cleaner noises (loud white noise). The model seems to perform fairly well, as evidenced by the predicted audio (and spectrograms) compared to the actual clean audio (and spectrograms).

Please let me know if you have any feedback. Thanks!

### CODE REVIEW

**Review by sanidhaya — APPROVED** [review]
The work is great, can you also push the model to Deployment branch before we push it to main so that i can merge it with deployment

**Review by ayushdh96 — APPROVED** [review]
Great work luke.

**Review by s1ashd0t — APPROVED** [review]
Great work! . One problem. To calculate the metrics i had to trim the output from model to the input in my eval class and the denoise function to successfully run evaluation on it. The difference between the two arrays is minute like 1/1000  but the eval functions run unless they are exactly same. So i did not perform full eval on it yet

TL;DR : I think the output from model is slightly longer
